### PR TITLE
fix(clustered): remove link to inaccessible google doc

### PR DESCRIPTION
**Pull request by jdockerty**
The Google doc link here is inaccessible to those outside of Influx, this is a leftover from before the document was ported to markdown in its early stages and subsequently moved to the docs site.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/clustered/admin/users/_index.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ draft: true
 To add or remove users, update the users list in `myinfluxdb.yml` file.
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/_index.md:13]:
```suggestion
To add or remove users, update the users list in the `myinfluxdb.yml` file.
```

File: content/influxdb/clustered/admin/users/_index.md [hunk 2]
```diff
@@ -13,6 +13,6 @@ draft: true
...
 The users list is found at spec.package.spec.admin.users.
 After updating the list, re-apply `myinfluxdb.yml`.
-See [Step 6: Deploy an InfluxDB cluster](https://docs.google.com/document/d/1UCmGtZS_OQe-gYOd0ASi0E2wFM9t3MOGwfDhxR1wtgg/edit#heading=h.m40zb08bbum1) for details on how to apply `myinfluxdb.yml`.
+See [Step 6: Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy) for details on how to apply `myinfluxdb.yml`.
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/_index.md:16]:
```suggestion
To learn how to apply `myinfluxdb.yml`, see [Step 6: Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy).
```

File: content/influxdb/clustered/admin/users/_index.md [hunk 3]
```diff
@@ -13,6 +13,6 @@ draft: true
...
 Once the `myinfluxdb.yml` has been applied, it will take a couple minutes for the updates to roll out.
 When complete, any new users will have been added and any removed users will have been deleted.
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/_index.md:18]:
```suggestion
After `myinfluxdb.yml` has been applied, updates take a couple of minutes to complete.
When the updates are finished, new users will have been added, and removed users will have been deleted.```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

**Comment by jstirnaman** [comment]
A few grammar suggestions.

File: content/influxdb/clustered/admin/users/_index.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ weight: 101
...
-To add or remove users, update the users list in `myinfluxdb.yml` file.
+To add or remove users, update the users list in the `myinfluxdb.yml` file.
 The users list is found at spec.package.spec.admin.users.
 After updating the list, re-apply `myinfluxdb.yml`.
-See [Step 6: Deploy an InfluxDB cluster](https://docs.google.com/document/d/1UCmGtZS_OQe-gYOd0ASi0E2wFM9t3MOGwfDhxR1wtgg/edit#heading=h.m40zb08bbum1) for details on how to apply `myinfluxdb.yml`.
-Once the `myinfluxdb.yml` has been applied, it will take a couple minutes for the updates to roll out.
-When complete, any new users will have been added and any removed users will have been deleted.
+To learn how to apply `myinfluxdb.yml`, see [Step 6: Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy).
+After `myinfluxdb.yml` has been applied, updates take a couple of minutes to complete.
+When the updates are finished, new users will have been added, and removed users will have been deleted.```
```
Comment (sanderson) [inline content/influxdb/clustered/admin/users/_index.md:18]:
```suggestion
To learn how to apply `myinfluxdb.yml`, see [Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy).
After `myinfluxdb.yml` has been applied, updates take a couple of minutes to complete.
When the updates are finished, new users will have been added, and removed users will have been deleted.
```

**Review by sanderson — APPROVED** [review]

**Pull request merged**